app/src/main/java/fr/neamar/kiss/dataprovider/ContactProvider.java

```diff
@@ -28,11 +28,13 @@ public ContactProvider(final Context context) {
         int matchPositionStart;
         int matchPositionEnd;
         String contactNameNormalized;
+        String phoneNormalized;
 
         final String queryWithSpace = " " + query;
         for (ContactPojo contact : pojos) {
             relevance = 0;
             contactNameNormalized = contact.nameNormalized;
+            phoneNormalized = contact.phone.replaceAll("[-.(): ]","");;
```

Comment:
for performance reason, it would probably be best to compute this normalizedPhone only once on LoadContacts instead, cause the regexp will be slow on devices with lots of contacts ;)